scripts/gdb/lvglgdb/lvgl.py

```diff
@@ -196,12 +164,12 @@ def draw_units(self):
             yield unit
             unit = unit["next"]
 
-def set_lvgl_instance(lv_global: gdb.Value):
+def set_lvgl_instance(lv_global: Optional[Value]):
```

Comment:
gdb.Value should be accepted and processed correspondingly.

Reply:
`gdb.Value` can be processed normally. It is recommended to pack it with `lvgl.Value` in advance before putting it in.

Reply:
```suggestion
def set_lvgl_instance(lv_global: Optional[gdb.Value, Value]):
```